src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -365,10 +365,11 @@ public Auth exchangeRefreshToken(String refreshToken) throws SmartcarException {
    *
    * @throws SmartcarException when the request is unsuccessful
    */
-  public boolean isCompatible(String vin) throws SmartcarException {
+  public boolean isCompatible(String vin, String scopes) throws SmartcarException {
```

Comment:
We would need to overload this function.
```java
public boolean isCompatible(String vin)
public boolean iscompatible(String vin, String scope)
```

And then write tests for both.

Reply:
^yep